### PR TITLE
Update some modules to mach styles

### PR DESCRIPTION
**Pull request by rodriciru**
| Questions         | Answers
| ------------------| -------------------------------------------------------
| Description?      | Some modules dosent have BS5 style
| Type?             |  improvement
| BC breaks?        |no
| Deprecations?     | no
| Fixed ticket?     | Fixes nothing
| How to test?      | Test it yourself.
| Possible impacts? | No.

### CODE REVIEW

File: modules/ps_newproducts/views/templates/hook/ps_newproducts.tpl [hunk 1]
```diff
@@ -23,13 +23,14 @@
...
+    <div class="container">
+        {include file="components/section-title.tpl" title={l s="New products" d="Shop.Theme.Catalog"}}
+        {include file="catalog/_partials/productlist.tpl" products=$products cssClass="row" productClass="col-6 col-lg-4 col-xl-3"}
+        <div class="new-products-footer text-center">
+            <a class="all-product-link btn btn-outline-primary" href="{$allNewProductsLink}">
+                {l s='All new products' d='Shop.Theme.Catalog'}<i class="material-icons">&#xE315;</i>
+            </a>
+        </div>
+    </div>
+</section>
```
Comment (NeOMakinG) [inline modules/ps_newproducts/views/templates/hook/ps_newproducts.tpl:36]: Missing empty line

You can configure your IDE so it adds empty lines automatically at the end of the file :)

**Comment by NeOMakinG** [comment]
Just 2 little comments, overwise looks fine to me!

File: modules/ps_newproducts/views/templates/hook/ps_newproducts.tpl (hunk 1 above)
Comment (rodriciru) [reply modules/ps_newproducts/views/templates/hook/ps_newproducts.tpl]: editorconfig maybe?

**Review by NeOMakinG — APPROVED** [review]

**Comment by NeOMakinG** [comment]
Thanks @rodriciru 🚀

**Comment by rodriciru** [comment]
You welcome

**Pull request merged**